eng/common/scripts/Package-Properties.ps1

```diff
@@ -92,13 +91,13 @@ function Get-PkgProperties
     foreach ($directory in $directoriesPresent)
     {
         $pkgDirectoryPath = Join-Path $serviceDirectoryPath $directory.Name
-        if ($ExtractPkgProps)
+        if ($GetPackageInfoFromRepoFn)
         {
-            $pkgProps = &$ExtractPkgProps -pkgPath $pkgDirectoryPath -serviceName $ServiceName -pkgName $PackageName
+            $pkgProps = &$GetPackageInfoFromRepoFn -pkgPath $pkgDirectoryPath -serviceName $ServiceName -pkgName $PackageName
         }
         else
         {
-            Write-Error "The function '${ExtractPkgProps}' was not found."
+            Write-Error "The function '${GetPackageInfoFromRepoFn}' was not found."
```

Comment:
This will say `The function '' was not found.` I would change it to `"The function Get-${Language}-PackageInfoFromRepo was not found."`